Examples/StockQuoteConnector_final.html

```diff
@@ -31,26 +34,63 @@
     return num < 9 ? "0" + num.toString() : num.toString();
   }
 
+  function getTickersArrayFromConnectionData(connectionData) {
+    var tickersArray = connectionData.split(',');
+    
+    // Remove whitespace from each ticker string
+    return tickersArray.map(function(obj) {
+      return obj.trim(); 
+    });
+  }
+
   //
   // Connector definition
   // 
-
   var myConnector = tableau.makeConnector();
 
-  myConnector.getColumnHeaders = function() {
-    var fieldNames = ['Ticker', 'Day', 'Close'];
-    var fieldTypes = ['string', 'date', 'float'];
-    tableau.headersCallback(fieldNames, fieldTypes); // tell tableau about the fields and their types
+  myConnector.getSchema = function(schemaCallback) {
+    var tickersArray = getTickersArrayFromConnectionData(tableau.connectionData);
+    
+    var schema = [];
+    _.forEach(tickersArray, function(ticker) {
+      var col1 = { id: "Ticker", dataType: "string"};
+      var col2 = { id: "Day", dataType: "date"};
+      var col3 = { id: "Close", dataType: "float"};
+      var cols = [col1, col2, col3];
+      
+      var tableInfo = { 
+        id: ticker,
+        columns: cols
+      };
+      
+      schema.push(tableInfo);
+    });
+   
+    schemaCallback(schema); // tell tableau about the fields and their types
   };
    
-  myConnector.getTableData = function(lastRecordToken) {
+  myConnector.getData = function(tables, doneCallback) {
```

Comment:
Commenting on the API overall, not just this example...  I wish we were more intentional with the dependencies that we inject on these main callbacks.  For instance, we're taking in a `TableCollection` instance here, but in the `getSchema` callback, we're not taking in anything, even though there might be opportunity to take in the connection details.

Or maybe it makes sense to keep the signatures for all of these callbacks the same (just a "done" callback), but open up more methods on the `tableau` object to get the stuff we need (`tableau.getConnectionInfo()`, `tableau.getTableCollections()`).